packages/expo-updates/android/src/androidTest/java/expo/modules/updates/loader/FileDownloaderManifestParsingTest.kt

```diff
@@ -24,14 +24,18 @@ import okhttp3.Response
 import okhttp3.ResponseBody.Companion.toResponseBody
 import org.junit.Assert
 import org.junit.Test
+import kotlinx.coroutines.test.runTest
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
 import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4ClassRunner::class)
 class FileDownloaderManifestParsingTest {
-  @Test
-  fun testManifestParsing_JSONBody() {
-    val context = InstrumentationRegistry.getInstrumentation().targetContext
+  @get:Rule
+  val temporaryFolder = TemporaryFolder()
```

Comment:
nit: since we replace android context with the `TemporaryFolder`. maybe we can move this test into unit test (rather than instrumentation test). that would be faster. can do this in a separate pr